torchci/lib/bot/verifyDisableTestIssueBot.ts

```diff
@@ -204,7 +205,9 @@ export default function verifyDisableTestIssueBot(app: Probot): void {
     const target = parseTitle(title);
     const platforms = parseBody(body!);
     const username = context.payload["issue"]["user"]["login"];
-    const authorized = await hasWritePermissions(context, username);
+    const authorized =
+      username === pytorchBotName ||
```

Comment:
Should we use id, rather than name here? (As we can change name later, but id is fixed)
(Alternatively, we can give bot triage permissions)